include/disccord/rest/api_client.hpp

```diff
@@ -224,6 +225,23 @@ namespace disccord
 
                     pplx::task<void> request(route_info& route, const pplx::cancellation_token& token = pplx::cancellation_token::none());
 
+                    pplx::task<web::json::value> request_raw_json(route_info& route, const pplx::cancellation_token& token = pplx::cancellation_token::none());
```

Comment:
its fine to have it as a model, just was hesitant since it is only one key. but it'll make things cleaner for the request interface, so might as well make them models